### ISSUE TITLE
Exiting game seems to disable running core in UI

### ISSUE THREAD

**Issue by georaldc** [libretro/RetroArch#2995]
I noticed that after loading any core, running a game and closing the game, the loaded core goes away (lower left status message will say "No Core"). The core still seems to be active though because you could go in and load another supported game and it would run. The UI seems to think otherwise because of the No Core message and the file browser not listing files (other than compressed zips) the running core accepts (do the same with a psx core for example and you won't find any ISO or cue files listed anymore)

**Comment by twinaphex** [comment]
Yes, that is the current design. If we add another option like 'Close Core' and make it separate from 'Close Content', users would just complain about there being too many options.

**Comment by georaldc** [comment]
But it doesn't fully close the core because like I said, selecting a new game that's compatible with the last core that was loaded would work (and choosing a non compatible game crashes the app). IIRC in previous versions, the loaded core continued to stay active even after exiting out of a game. I could be wrong though

**Comment by twinaphex** [comment]
list the exact steps which you describe as 'selecting a new game' so that I can reproduce this and we don't get confused talking about an entirely different issue.

**Comment by georaldc** [comment]
Let's use a NES core like fceumm and 2 games for example, 1 .nes file and 1 .zip file containing a .nes rom

1. Load core and select fceumm. Bottom left should now show (using latest nightly) 1.3.4 - FCEUmm
2. Load content > Select File and browse to NES games directory. You should see both .nes and .zip files. Load either.
3. After game loads, return to quick menu and select "Close" to close game.
4. Bottom left will now show "1.3.4 - No Core"
5. Select Load Content again and navigate to your NES games folder. You'll notice the .nes file not showing anymore (probably because the fceumm core isn't loaded anymore) but the .zip file should still be selectable. It should also load just fine, despite retroarch saying that no core was loaded.

**Comment by junkdeck** [comment]
Is it possible to change this behavior in the config files somehow, or even just make "Run" the default option instead of "Information" when selecting a core?

I doubt people would complain much about options though, they just allow you to tailor software to your own preferences. There's already a way to hide menu items you don't use, so why not include an option for all three and make the "Close Content & Core" the default?

**Comment by fr500** [comment]
No. It will reintroduce a bunch of problems.

**Comment by junkdeck** [comment]
that's a no to default of "run" as well?

**Comment by fr500** [comment]
No, that's a bug and should be fixed.

**Comment by zoltanvb** [comment]
This should not be an issue nowadays, removing the core is a two-step procedure now: close content, then unload core.
Core selection also works reasonably, I can not find any specific place where "run" would not be the default.
